node_modules/@okta/courage/package.json

```diff
@@ -8,7 +8,6 @@
   "_from": "@okta/courage@1.20.0",
   "_id": "@okta/courage@1.20.0",
   "_inCache": true,
-  "_installable": true,
```

Comment:
I'm using npm 3.9.1. maybe someone else ever upgrade courage using newer version of npm before?
(but travis didn't complain about this config)